### PR TITLE
feat: support fzf-lua as a folder picker

### LINKED ISSUE #15: Support `fzf-lua` as a folder picker
Hey, I am using `ibhagwan/fzf-lua` instead of `junegunn/fzf.vim`. Sounds like it does not works the same ? 

after setting:

```vim
let g:himalaya_folder_picker = 'fzf'
```

I get the bellow  error:
```
Error detected while processing function <lambda>71[1]..<SNR>39_on_exit[9]..<lambda>68[1]..<SNR>36_on_json_data[1]..<lambda>67[1]..<SNR>35_op
en_picker[13]..himalaya#domain#folder#pickers#fzf#select:
line    1:
E117: Unknown function: fzf#run
```

thanks for your insights

### PR DESCRIPTION
**Pull request by parisni**
fzf-lua is the default picker in lazyvim distro.

fixes #15

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- support for `fzf-lua` as a folder picker
+
+### Added
```
Comment (soywod) [inline CHANGELOG.md:14]: Duplicate section

**Review by soywod — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution, much appreciated :pray: I just had a minor change request on the changelog, otherwise we can merge!

File: CHANGELOG.md (hunk 1 above)
Comment (parisni) [reply CHANGELOG.md]: fixed

**Review by soywod — APPROVED** [review]

**Comment by soywod** [comment]
Thank you, sorry for the delay!

**Pull request merged**